test/navigation.spec.js

```diff
@@ -897,7 +897,6 @@ module.exports.describe = function({testRunner, expect, playwright, MAC, WIN, FF
       await page.$eval('iframe', frame => frame.remove());
       const error = await navigationPromise;
       expect(error.message).toContain('frame was detached');
-      expect(error.stack).toContain('Frame.goto')
```

Comment:
this was very important for debugging.

Reply:
I looked at the stacks without it and it does look very straightforward. Could you give an example that was good and is now puzzling?

Reply:
It's not as bad as it once was, because the async stack traces help us at least get the client code that called this. But lacking the playwright method you called in the stack trace is not great.